packages/module/patternfly-docs/content/examples/filterSidePanel.css

```diff
@@ -1,50 +1,46 @@
 .filter-panel-pf {
```

Comment:
Ideally this whole thing would be a `<Form>`. I think the markup would be different since currently each block of checkboxes is in its own `<form>` HTML element  but it should give exactly the styling needed. Here's a rough codesandbox https://codesandbox.io/p/sandbox/gallant-bouman-wcjx32?file=%2Findex.tsx%3A31%2C15. The margins and everything in the v6 `<Form>` matched the figma design for this panel as far as I could tell.